wagtail/admin/static_src/wagtailadmin/js/vendor/tag-it.js

```diff
@@ -222,8 +222,18 @@
                     }
                 });
             }
-
             // Events.
+            this.tagInput.bind('paste', function (event) {
```

Comment:
Could we make it more obvious (with a comment) that this is our custom patching of a vendored file?